core/src/main/scala/org/apache/spark/scheduler/DAGScheduler.scala

```diff
@@ -769,7 +790,9 @@ class DAGScheduler(
     }
   }
 
-  /** Called when stage's parents are available and we can now do its task. */
+  /**
+   * Called when stage's parents are available and we can now do its task.
```

Comment:
The changes here that just make scaladoc into a multiline comment don't do anything .. it's still valid on one line. I think the other changes look fine, and this is a no-op at best, but personally would not change these lines if there is no functional change.